src/test/java/org/jboss/xavier/analytics/rules/workload/inventory/ReasonableDefaultsTest.java

```diff
@@ -0,0 +1,111 @@
+package org.jboss.xavier.analytics.rules.workload.inventory;
+
+import org.jboss.xavier.analytics.pojo.output.workload.inventory.WorkloadInventoryReportModel;
+import org.jboss.xavier.analytics.rules.BaseTest;
+import org.jboss.xavier.analytics.test.Utils;
+import org.junit.Assert;
+import org.junit.Test;
+import org.kie.api.io.ResourceType;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ReasonableDefaultsTest extends BaseTest {
+
+    public ReasonableDefaultsTest()
+    {
+        super("/org/jboss/xavier/analytics/rules/workload/inventory/ReasonableDefaults.drl", ResourceType.DRL);
+    }
+
+    private void checkLoadedRulesNumber()
+    {
+        Utils.checkLoadedRulesNumber(kieSession, "org.jboss.xavier.analytics.rules.workload.inventory", 3);
+    }
+
+    @Test
+    public void testDatacenterFieldNullValueShouldFireRule()
+    {
+        checkLoadedRulesNumber();
```

Comment:
you can extract this check with a @Before method

Reply:
See above.